app/views/govuk_publishing_components/components/_transition_countdown.html.erb

```diff
@@ -0,0 +1,33 @@
+<%
+  countdown_clock = GovukPublishingComponents::AppHelpers::CountdownHelper.new
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+
+  heading_level ||= 2
+  title ||= nil
+  text ||= nil
+  url ||= nil
+  data_attributes ||= {}
+  css_classes = %w(gem-c-transition-countdown)
+  css_classes << 'gem-c-transition-countdown--cta' if url
+  css_classes << 'govuk-link' if url
+%>
+
+<% countdown = capture do %>
+  <%= content_tag(shared_helper.get_heading_level, title, class: "gem-c-transition-countdown__title") if title %>
+  <% if countdown_clock.show? %>
+    <%= tag.p class: "gem-c-transition-countdown__countdown" do %>
+      <%= tag.span countdown_clock.days_left.first, class: "gem-c-transition-countdown__countdown-number" %><%= tag.span countdown_clock.days_left.last, class: "gem-c-transition-countdown__countdown-number" %> <%= tag.span countdown_clock.days_text, class: "gem-c-transition-countdown__countdown-text" %>
```

Comment:
FWIW - I noticed on the transition page that screenreaders read this out disjointedly so "44 days to go" reads "4" "4" this issue might replicate here

Reply:
Do you remember which screen reader? Voiceover seems to work fine with this implementation (which is not the same as the one we currently have on `/transition`)

<img width="646" alt="Screenshot 2020-11-18 at 19 47 34" src="https://user-images.githubusercontent.com/788096/99581170-4d362180-29d8-11eb-86f0-d6ddb8dd832b.png">

Reply:
Ah, sorted 👍